examples/apollo/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // eslint-disable-next-line import/order
 import { enableTracing } from '../../../packages/node/src';
```

Comment:
🤔 We could add `@envy/node` as a dependency in the `package.json` to make this look a bit cleaner

Reply:
That didn't work when I'd tried it before, since it tries to install that package from npm rather than from the workspace (since this is in the `examples` folder and not `packages`).

Not sure whether CB fixed that since I wrote that import though.

Do you have it done that way for your express app?

Reply:
I change some configuration to get it to work an I updated `apollo` and `express` to use it in this PR https://github.com/FormidableLabs/envy/pull/26